src/main/java/soc/client/SOCPlayerClient.java

```diff
@@ -194,7 +177,7 @@
      * i18n text strings in our {@link #cliLocale}.
      * @since 2.0.00
      */
-    final soc.util.SOCStringManager strings;
+    final soc.util.SOCStringManager strings; // TODO: not a very good name
```

Comment:
Since it's widely used inside of method calls, this name has to be short but obvious (not an abbreviation like sm). I'm open to better suggestions, but it seems OK as-is right now

Reply:
Yeh, I couldn't come up with anything at the time, I just made a comment so we could come back to it.